spec/models/step_by_step_page_spec.rb

```diff
@@ -134,6 +137,68 @@
     }.to raise_error(ActiveRecord::RecordNotUnique)
   end
 
+  describe 'broken link checking' do
+    let(:step_by_step_with_step) { create(:step_by_step_page) }
+
+    it 'checks if a link check was requested and there are no broken links' do
```

Comment:
@leenagupte - this test title doesn't seem to represent the actual test, I think the title for this and the one below need to be swapped

Reply:
Good spot! Thanks! Will change and fix-up.